app/views/supervisors/index.html.erb

```diff
@@ -30,26 +29,13 @@
         </div>
         </span>
         <span class="ml-5">
-            <% if @show_all %>
-                <%= link_to supervisors_path, class: "btn-sm main-btn primary-btn hide-deactivated" do %>
-                  <i class="lni lni-checkmark mr-10"></i>
-                Hide deactivated
-              <% end %>
-            <% else %>
-                <%= link_to supervisors_path(all: "true"), class: "btn-sm main-btn success-btn show-deactivated" do %>
-                  <i class="lni lni-bulb mr-10"></i>
-                Show deactivated
-              <% end %>
```

Comment:
Is there a reason why this is getting removed, or was it not really in use?

Reply:
> Is there a reason why this is getting removed, or was it not really in use?

This was for the previous backend based filter for deactivated. Because we're using DataTable now, we don't need this button, as the filter is built in there.